src/common/services/craftApi.js

```diff
@@ -23,21 +23,16 @@ module.exports = {
           craftType,
           craftBase,
         }),
-      }, (error, response, body) => {
+      }, (error, _response, body) => {
```

Comment:
Linter complains of an unused parameter, so the underscore is added to bypass that.